ansible/roles/web/tasks/rabbitmq.yml

```diff
@@ -2,12 +2,13 @@
 # Install RabbitMQ
 # see http://www.rabbitmq.com/install-debian.html
 
-- name: Add RabbitMQ bintray repository key
+- name: Add RabbitMQ and Erlang key
   become: true
   apt_key:
     url: "{{item}}"
   with_items:
-    - 'https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc'
+    - 'https://packagecloud.io/rabbitmq/rabbitmq-server/gpgkey'
+    - 'https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc'
```

Comment:
Please revert those changes as discussed. If bintray repositories don't work for you we need to find another solution in another PR.